process/code/UJS.java

```diff
@@ -0,0 +1,191 @@
+
+import java.awt.CardLayout;
+import java.util.ArrayList;
+import java.util.Random;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+
+public class UJS extends javax.swing.JFrame {
+
+    protected static double goldPricePerGram = 175 + (200 - 175) * new Random().nextDouble();
+    protected static double silverPricePerGram = 2.63 + (5 - 2.63) * new Random().nextDouble();
+    protected static double diamondPricePerGram = 1800 + (2100 - 1800) * new Random().nextDouble();
+    protected static double jadePricePerGram = 50 + (60 - 50) * new Random().nextDouble();
+    protected static double platinumPricePerGram = 240 + (250 - 240) * new Random().nextDouble();
```

Comment:
Question, are these the set prices of the material types based of current market costs??